lib/buffer.js

```diff
@@ -637,6 +637,28 @@ Buffer.prototype.includes = function includes(val, byteOffset, encoding) {
   return this.indexOf(val, byteOffset, encoding) !== -1;
 };
 
+// Limit currently not implemented; reserved
+Buffer.prototype.split = function split(sep, limit) {
+  if (!limit)
+    limit = null;
+  if (!sep)
+    return [this];
```

Comment:
I think the docs should note that in this case the resulting `Buffer` is not a copy, as in all other situations? Alternatively: Could maybe all returned `Buffer`s be slices/subarrays? That might be more performant, and users could still create copies if they want to.